packages/mask/src/index.js

```diff
@@ -163,9 +163,9 @@ export function buildUp(template, input) {
     return output
 }
 
-function formatMoney(input, delimeter = '.', thousands) {
+export function formatMoney(input, delimeter = '.', thousands) {
     thousands = (delimeter === ',' && thousands === undefined)
-        ? '.' : ','
+        ? '.' : thousands || ','
```

Comment:
Simple and clean.

I'd probably do this as 

```js
thousands = thousands ?? delimiter === ',' ? '.' : ','
```
This would allow it to short circuit on a provided value quicker, and then also the ternary is clearer.

Maybe also fix the misspelling of `delimeter` while we're here.

Reply:
Hi Eric, thank you for the nice feedback. Indeed the `??` operator is clearer. I went ahead and renamed the delimiter argument.